packages/jetbrains-plugin/src/test/kotlin/com/mongodb/jbplugin/fixtures/components/MdbJavaEditorToolbarFixture.kt

```diff
@@ -30,7 +30,16 @@ class MdbJavaEditorToolbarFixture(
         remoteComponent,
     ) {
     val dataSources: ComboBoxFixture
-        get() = find(byXpath("//div[@class='ComboBox']"))
+        get() = find(byXpath("//div[@class='DataSourceComboBox']"))
```

Comment:
Was having a concrete XPath also a motivation to refactor the MDBJavaEditorToolbar into separate comboboxes?

Reply:
To be fair no, the MDBJavaEditorToolbar was already pretty big with one single dropdown implementation and I thought that refactoring was going to make easier to add new actions later. Accessing by XPath simplifies a lot testing but I usually don't write the XPath myself :D I go to the IntelliJ inspector and take it from there.

Reply:
Understood - thank you for the context Kevin 🙌